requirements/static/pkg/py3.6/linux.txt

```diff
@@ -57,6 +59,8 @@ more-itertools==5.0.0
     #   jaraco.functools
 msgpack==1.0.2
     # via -r requirements/base.txt
+packaging==21.3
```

Comment:
Did you confirm the licensing on looseversion, packaging and pyparsing is okay to include in this project?

Reply:
Licenses:
packaging is Apache 2.0 and BSD clause 2
looseversion is Python Software Foundation License (PSF-2.0)
pyparsing is MIT License